docs/model_fields.rst

```diff
@@ -695,6 +695,45 @@ Or with attribute assignment to a model::
         any conflicts.
 
 
+.. _enum-fields:
+
+-----------
+Enum Fields
+-----------
+
+You can take advantage of MySQL's ``ENUM`` type through the ``EnumField``
+provided by django-mysql. Using ``ENUM`` provides the benefit of storing
+recurring strings in a very space efficient manner. MySQL will store the
+enumeration value in each row as an integer value and return results as
+the corresponding strings in queries.
+
```

Comment:
you should

1. add docs links
2. add class definition

```
 .. class:: EnumField(choices, **kwargs)
```

See below in the file for resizable binary/string fields examples.